.travis.yml

```diff
@@ -17,7 +17,6 @@ before_install: git checkout $(git branch --contains $(git rev-parse HEAD)|tail
 
 jdk:
   - openjdk8
-  - openjdk11
```

Comment:
We don't use openjdk11 testing in any of our other applications. It was causing an error in travis for all PRs